Stripe/ApplePay/STPAPIClient+ApplePay.m

```diff
@@ -17,6 +17,55 @@ - (void)createTokenWithPayment:(PKPayment *)payment completion:(STPTokenCompleti
     [self createTokenWithData:[self.class formEncodedDataForPayment:payment] completion:completion];
 }
 
++ (NSString * _Nonnull)queryStringForAddress:(ABRecordRef _Nullable)addressRef {
+    NSMutableDictionary *params = [NSMutableDictionary dictionary];
+    __block NSString *queryString = [NSString string];
+
+    NSMutableCharacterSet *set = [[NSCharacterSet URLQueryAllowedCharacterSet] mutableCopy];
```

Comment:
nitpick, but we're creating this character set twice. can we add an `allowedCharacters:(NSCharacterSet *)` param?